.travis.yml

```diff
@@ -54,7 +54,7 @@ before_script:
 
 # Default script for release builds.
 script:
-- cargo build --release --locked --all-targets
+- cargo build -vv --release --locked --all-targets
```

Comment:
You could drop this, the cache is already filled and this is no longer necessary.

Reply:
We have to empty  the cache sometimes for whatever reasons.
And when we upgrade v8, the build takes a long time.
So I'd rather keep it for that reason; it would be nice though if we could have *some* output but not as much as we get with -vv.

Reply:
We should leave it if removing it may cause travis failures. We do rebuild V8 fairly regularly.